src/util/ChartUtils.ts

```diff
@@ -165,10 +166,18 @@ export const getMainColorOfGraphicItem = (item: ReactElement) => {
   return result;
 };
 
-interface FormattedGraphicalItem {
-  props: any;
+export interface FormattedGraphicalItem {
+  props: {
+    sectors?: ReadonlyArray<any>;
+    data?: ReadonlyArray<any>;
+  };
   childIndex: number;
-  item: any;
+  item: ReactElement<{ legendType?: LegendType; hide: boolean; name?: string; dataKey: unknown }>;
```

Comment:
Love the usage on unknown instead of any - but we could also use the DataKey type here, right? 
from `src/util/types.ts`
``` 
export type DataKey<T> = string | number | ((obj: T) => any);
```

Reply:
I have seen in multiple occasions in this codebase that variable with same name is completely different time so I am hesitating to assume. I will update the type if I track down that it's actually passing the `DataKey`.

Reply:
Fair enough 😅